tests/functional/scenario1/DatadirScenarioTest.php

```diff
@@ -0,0 +1,192 @@
+                        'business_schema' => [
+                            'schema_name' => 'my_dwh_schema',
+                        ],
+                    ],
+                ],
+            ],
+            'create-schema-2' => [
+                [
+                    'parameters' => [
+                        'master_host' => getenv('HOST'),
+                        'master_user' => getenv('USER'),
+                        '#master_password' => getenv('PASSWORD'),
+                        'master_database' => getenv('DATABASE'),
+                        'warehouse' => getenv('WAREHOUSE'),
+                        'business_schema' => [
+                            'schema_name' => 'my_dwh_schema2',
+                        ],
+                    ],
+                ],
+            ],
+            'create-user-user1' => [
+                [
+                    'parameters' => [
+                        'master_host' => getenv('HOST'),
+                        'master_user' => getenv('USER'),
+                        '#master_password' => getenv('PASSWORD'),
+                        'master_database' => getenv('DATABASE'),
+                        'warehouse' => getenv('WAREHOUSE'),
+                        'user' => [
+                            'email' => 'user1@keboola.com',
+                            'business_schemas' => ['my_dwh_schema'],
+                            'disabled' => false,
+                        ],
+                    ],
+                ],
+            ],
+            'create-user-user2' => [
+                [
+                    'parameters' => [
+                        'master_host' => getenv('HOST'),
+                        'master_user' => getenv('USER'),
+                        '#master_password' => getenv('PASSWORD'),
+                        'master_database' => getenv('DATABASE'),
+                        'warehouse' => getenv('WAREHOUSE'),
+                        'user' => [
+                            'email' => 'user2@keboola.com',
+                            'business_schemas' => ['my_dwh_schema', 'my_dwh_schema2'],
+                            'disabled' => false,
+                        ],
+                    ],
+                ],
+            ],
+        ];
+    }
+
+    public function provideConfigs(): array
+    {
+        return self::getTestConfigs();
+    }
+
+    /**
+     * @dataProvider provideConfigs
+     */
+    public function testDatadir(array $config): void
+    {
+        self::$logger->log(Logger::DEBUG, $this->getDataSetAsString());
+
+        $specification = new DatadirTestSpecification(null, 0, null, null, null);
+
+        $tempDatadir = $this->getTempDatadir($specification);
+
+        file_put_contents($tempDatadir->getTmpFolder() . '/config.json', json_encode($config));
+
+        $process = $this->runScript($tempDatadir->getTmpFolder());
+
+        $this->assertMatchesSpecification($specification, $process, $tempDatadir->getTmpFolder());
```

Comment:
Testuje to jenom že to skončí success nebo i něco víc?

Reply:
Myslel jsem spíš tu druhou variantu tzn. vykoušet jestli má přístup kam je potřeba. Takový to co jsem dělal ručně když jsem dělal review na minulé PRs. Myslím že by to stálo za pokus, ale klidně to zkus až nové feature.

Reply:
No, tam je jediný problém s tím vygenerovaným heslem. Ale zkusím.